_articles/system76-software.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: article
+title: Install System76 Software in other OSes
+description: >
+    Learn what software is needed to use other OSes and install them
+keywords:
+  - system76
+  - support
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: hardware-drivers
+
+---
+
+## Arch
+
+Be sure to install the <u>System76 Driver</u> first and the steps to do that are [here](/articles/system76-driver). 
```

Comment:
Make this two sentences:

Be sure to install the <u>System76 Driver</u> first. The steps to do that are [here]